developers.properties

```diff
@@ -0,0 +1 @@
+nikita.lebedev@limechain.tech=Nikita Lebedev
```

Comment:
Nikita is no longer maintaining the SDK.

Reply:
Until now, this was still configured (moved here from another place)

What should we put here?

In other places it currently looks like this, but also there I am not sure if it is right:
https://github.com/hashgraph/hedera-services/blob/14255-reusable-gradle-config/platform-sdk/developers.properties

The information ends up here: https://repo1.maven.org/maven2/com/hedera/hashgraph/sdk/2.45.0/sdk-2.45.0.pom
(`<developers>` section)


Should we put something like?

```
<some-team>@hiero.org=Some Team
```

 Maybe there is some general pattern you want to follow in the future? @hendrikebbers maybe?

Reply:
After discussing with Hendrik, we concluded to remove this, if you have no objections.

Instead, we add a generic `<developer>The Hiero Team</developer>`.

Otherwise this it an additional place to maintain who develops what, which is quickly outdated (as is is here right now).